docker-compose.yml

```diff
@@ -30,8 +25,6 @@ services:
       - "./kibana/config/:/etc/kibana"
   service-registry:
     build: service-registry/
-    ports:
-      - "8761:8761"
```

Comment:
I agree with you to close the the ports, but from my point of view eurekas port is to much. For the demonstration I can use the eureka console to show the registered services.